src/module/scene/helpers.ts

```diff
@@ -42,4 +44,9 @@ const checkAuras = foundry.utils.debounce(async function (this: ScenePF2e): Prom
     lock.release();
 }, 100);
 
-export { checkAuras };
+/** Returns true if the given actor type is valid for token size linking and autoscaling */
+function isActorTypeSizeLinkable(type: string): boolean {
+    return tupleHasValue(CREATURE_ACTOR_TYPES, type) || type === "vehicle";
+}
```

Comment:
A bit overwrought as a helper: can this just be a set (even if only a `Set<string>`)?

Reply:
Can be, I'll swap it over.

Reply:
Its a set now. Since its a set it can go in values, so back to actor/values.ts it goes, avoiding the cyclical reference.